website/docs/release_notes.md

```diff
@@ -8,6 +8,187 @@ import ReactPlayer from 'react-player'
+# [info] Discovered 478 classes and 1912 methods after optimization
+# [info] Optimizing (debug mode) (403 ms)
+# [info] Produced 9 LLVM IR files
+# [info] Generating intermediate code (368 ms)
+# [info] Compiling to native code (1565 ms)
+# [info] Linking with [pthread, dl]
+# [info] Linking native code (immix gc, none lto) (83 ms)
+# [info] Postprocessing (0 ms)
+# [info] Total (3625 ms)
+# Hello from Scala Native 0.5.5!
+```
+
+Added by [@Gedochao](https://github.com/Gedochao) in [#3117](https://github.com/VirtusLab/scala-cli/pull/3117).
+
+### (⚡️ experimental) Support for exporting to a Maven project
+It is now possible to export a Scala CLI project to Maven.
+
+```bash
+scala-cli export --script-snippet 'println("No need to create the pom.xml yourself!")' --mvn --power -o mvn-demo
+# Some utilized features are marked as experimental:
+#  - `export` sub-command
+#  - `--mvn` option
+# Please bear in mind that non-ideal user experience should be expected.
+# If you encounter any bugs or have feedback to share, make sure to reach out to the maintenance team at https://github.com/VirtusLab/scala-cli
+# Exporting to a maven project...
+# Exported to: ~/scala-cli-tests/mvn-demo
+cd mvn-demo
+mvn scala:run -DmainClass=snippet_sc
+# (...)
+# No need to create the pom.xml yourself!
+# [INFO] ------------------------------------------------------------------------
+# [INFO] BUILD SUCCESS
+# [INFO] ------------------------------------------------------------------------
+# [INFO] Total time:  2.589 s
+# [INFO] Finished at: 2024-08-22T12:08:36+02:00
+# [INFO] ------------------------------------------------------------------------
+```
+
+Added by [@yadavan88](https://github.com/yadavan88) in [#3003](https://github.com/VirtusLab/scala-cli/pull/3003).
+
+### Support for launching apps from dependencies without other inputs
+It is now possible to launch an app by just specifying its dependency, without the need to provide any source files.
+In such a case the build server will not be started, as there's no compilation to be done.
+There's also no need to specify the main class, as it's now being detected automatically in dependencies as well.
+Do note that explicitly calling the `run` sub-command is necessary here, as otherwise Scala CLI will default to the REPL.
+
+```bash
+scala-cli run --dep io.get-coursier:coursier-cli_2.13:2.1.10 -- version
+# 2.1.10
+```
+
+This can be used similarly to [Coursier's `cs launch`](https://get-coursier.io/docs/cli-launch).
+
+Added by [@kasiaMarek](https://github.com/kasiaMarek) in [#3079](https://github.com/VirtusLab/scala-cli/pull/3079).
+
+### (⚡️ experimental) JMH available in various commands and via `using` directives
+Some improvements have been done to the experimental support for JMH (Java Microbenchmark Harness).
+
+The `--jmh` and `--jmh-version` options can now be passed to a number of commands:
+- `compile`, so that a Scala CLI project with benchmarking can be compiled separately from being run.
```

Comment:
```suggestion
- `compile`, so that a Scala CLI project with benchmarking can be compiled separately from being run;
```
NIT, but I don't think `.` makes much sense there.